translations/fr.json

```diff
@@ -711,6 +726,7 @@
     "notifications": "Notifications",
     "notify": "Notifier",
     "off": "Désactivé",
+    "off-status-tree": "Arbre Status désactivé",
```

Comment:
Not 100% sure of the context, even after reading the english translation, but I don't think this is the wanted translation
```

Reply:
I get the context and we should translate it "En dehors de l'arbre Status", which means outside of the tree of derivation paths from the status seed :)

Reply:
Ok, that's similar to what I thought, but wanted to be sure. Thanks

Reply:
@jrainville @guylouis I've just updated the commit. Please take a look :)